### ISSUE TITLE
Tables and Charts show "No Data" when no data

### ISSUE THREAD

**Issue by ShanikaEdiriweera** [tharakanandakumara/sign#5]
Tables and Charts show "No Data" when no data from the fetch request.
- Class wise daily report
- Class wise monthly report
- Grade wise daily report
- Grade wise monthly report

**Comment by iamhabee** [comment]
hi @ShanikaEdiriweera can i work on this

**Comment by ShanikaEdiriweera** [comment]
@iamhabee Sure. Pls go on :100:

**Comment by ShanikaEdiriweera** [comment]
@iamhabee can I expect a PR soon? Or shall I unassign this and make open to others?

**Comment by iamhabee** [comment]
Yes am sorry am out of data for now I will definitely make a PR tomorrow

**Comment by iamhabee** [comment]
so i was able to fix it but having authentication error with aws to test what i did. please if there is any credentials i need to complete this please help me with it.
Thanks

**Comment by ShanikaEdiriweera** [comment]
This is the backend repo - https://github.com/ShanikaEdiriweera/student-attendance-system-backend Try to set it up.

Check the comments here ShanikaEdiriweera/student-attendance-system-backend#1
Have to add config files and run mongodb locally!

Then you can create an admin user for yourself using the /user API endpoint. check the sample request in samples

**Comment by iamhabee** [comment]
Am sorry for any inconvenience I might av caused on this project. I thought I will be able to make up with my personal challenge but I can't. Please forgive my attitude and thanks for the privilledge u give me

**Comment by ShanikaEdiriweera** [comment]
@iamhabee no issue.

So this issue is open for anyone for grab.

**Comment by darkeN64** [comment]
Hi, can i work on this ?

**Comment by ShanikaEdiriweera** [comment]
@darkeN64 sure.

**Comment by ShanikaEdiriweera** [comment]
@darkeN64 hey how is the progress on this?

**Comment by darkeN64** [comment]
Hello, I have a little problem... I set up the project by following "Setting up the project" instructions, but i have a problem with last step - "You may create an admin user to access the dashboard using the /user API endpoint of the backend."

How should i create admin user ? I tried putting /user on the end of the backend server url, but it returned only "Cannot GET /user". server is running, mongodb too, 
http://127.0.0.1:3000/ returns "{"message":"Welcome to RFID System."}"
http://127.0.0.1:3000/user returns "Cannot GET /user".

So... how can i create admin user ? Through GET or POST ? Or other solution ?

**Comment by darkeN64** [comment]
Finally I have successfully set up the project and seems like everything is working fine. How can I view this issue(which I am working on) ? I can't find any report which shows "no data".
